.rubocop.yml

```diff
@@ -37,6 +37,10 @@ Layout/EmptyLinesAroundArguments:
 Layout/SpaceInsideStringInterpolation:
   EnforcedStyle: no_space
 
+Naming/FileName:
+  Exclude:
+    - 'lib/rubocop-md.rb'
```

Comment:
This PR specifies `Exclude`, it may be better to change `Naming/FileName` cop to allow `lib/rubocop-md.rb` file name with the same name as `rubocop-md.gemspec` in the future.

For example, net-telnet gem uses the same `lib/net-telnet.rb` as `net-telnet.gemspec`.
https://github.com/ruby/net-telnet

Reply:
> it may be better to change Naming/FileName cop to allow lib/rubocop-md.rb file name with the same name as rubocop-md.gemspec in the future

Hm, interesting idea, I like it. 
p/s/ and thanks for your job 👍